packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -379,41 +387,56 @@ QUnit.module('MessageList', moduleConfig, () => {
             });
         });
 
-        QUnit.test('Scrollable should be scrolled to last message group after init', function(assert) {
+        QUnit.test('should be scrolled to last message after init', function(assert) {
+            const done = assert.async();
+
             this.reinit({
                 width: 300,
                 height: 600,
                 items: generateMessages(52)
             });
 
-            const scrollTop = this.scrollable.scrollTop();
+            setTimeout(() => {
```

Comment:
Is it really necessary to use setTimeout here? Doesn't component have options like onInit (or onContentReady XD), it doesn't fit?

Reply:
Yes, it's really needed. Without it, we can't handle the resizeObserver call properly